npm/package.json

```
@@ -8,7 +8,7 @@
     "postinstall": "node install.js"
   },
   "dependencies": {
-    "@electron/get": "^1.13.0",
+    "@electron/get": "^1.14.0",
```

suggestion: there was a bug in 1.14.0.

```suggestion
    "@electron/get": "^1.14.1",
```

---

Yes. Sorry, I forgot this PR existed.